documentation/assemblies/assembly-kafka-connect-external-configuration.adoc

```diff
@@ -11,14 +11,13 @@
 
 = Using external configuration and secrets
 
-Kafka Connect connectors are configured using an HTTP REST interface.
-The connector configuration is passed to Kafka Connect as part of an HTTP request and stored within Kafka itself.
+Connectors are created, reconfigured, and deleted using an HTTP REST interface unless `KafkaConnectors` are enabled for the Kafka Connect cluster. The connector configuration is passed to Kafka Connect as part of an HTTP request and stored within Kafka itself.
```

Comment:
I think a link to more documentation on the KafkaConnectors would be beneficial when the reader hasn't read about them yet.